.gitlab-ci.yml

```diff
@@ -54,6 +54,36 @@ tox:
     - !reference [.load_github_key, script]
     - tox -e py
 
+# used by release
+.python_versions:
+  parallel:
+    matrix:
+      - BUILD_PY_VERSION: [python3.6]
+      - RELEASE_PY_VERSION: [python3.6]
+
+release:
+  before_script:
+    - git fetch --unshallow
+
+get_version:
+  stage: test
+  tags:
+    - dind
+  image: ${BASE_CONTAINER_REGISTRY}/python3.8-builder:${BASE_CONTAINER_VERSION}
+  before_script:
+    - git fetch --unshallow
+  script:
+    - pip install importlib-metadata
+    - pip install --no-deps .
```

Comment:
Nice!!!

Hopefully, soon we'll do that for running tests too: `pip install .[dev]" in tox.

And even later on, maybe even for docker build, assuming the artifact was built and successfully pushed to nexus, `pip install indexd==version`.